lib/matplotlib/afm.py

```diff
@@ -304,47 +359,55 @@ def _parse_optional(fh):
         if key in optional:
             d[key] = optional[key](fh)
 
-    l = (d[b'StartKernData'], d[b'StartComposites'])
-    return l
+    return d[b'StartKernData'], d[b'StartComposites']
 
 
 def parse_afm(fh):
     """
-    Parse the Adobe Font Metics file in file handle *fh*. Return value
-    is a (*dhead*, *dcmetrics_ascii*, *dmetrics_name*, *dkernpairs*,
-    *dcomposite*) tuple where
-    *dhead* is a :func:`_parse_header` dict,
-    *dcmetrics_ascii* and *dcmetrics_name* are the two resulting dicts
-    from :func:`_parse_char_metrics`,
-    *dkernpairs* is a :func:`_parse_kern_pairs` dict (possibly {}) and
-    *dcomposite* is a :func:`_parse_composites` dict (possibly {})
+    Parse the Adobe Font Metics file in file handle *fh*.
+
+    Returns
+    -------
+    header : dict
+        A header dict. See :func:`_parse_header`.
+    cmetrics_by_ascii : dict
+        From :func:`_parse_char_metrics`.
+    cmetrics_by_name : dict
+        From :func:`_parse_char_metrics`.
+    dkernpairs : dict
```

Comment:
I think this 'd' in the front stood for dict, so it's probably not needed with the type now explicitly named.